components/html-block/demo/html-block.html

```diff
@@ -9,6 +9,60 @@
 		<script type="module">
 			import '../../demo/demo-page.js';
 			import '../html-block.js';
+			import { provideInstance } from '../../../mixins/provider-mixin.js';
+
+			let profileCardAccess;
```

Comment:
The code to support `@mentions` is basically copied from the LMS and cleaned up a bit.  I placed it here in this demo file for purposes of temporary demo only... the renderer for `@mentions` perhaps better belongs in [BrightspaceUILabs/user-profile-card](https://github.com/BrightspaceUILabs/user-profile-card), and BSI would pull in the domain specific renderers and provide them via `provideInstance`.